src/utility/Ex3.py

```diff
@@ -0,0 +1,230 @@
+import json
+import random
+import numpy as np
+import scipy.stats as stats
+
+class Pedestrian:
```

Comment:
Sadly we can't determine the number of pedestrians (aka `numOfAgents`) since it's given in the `input.json`. But we could add a field to register number of each type.